src/index.js

```diff
@@ -55,14 +55,16 @@ function withPlugins(
 
 const formatWithCursor = withPlugins(core.formatWithCursor);
 
-async function format(text, opts) {
-  const { formatted } = await formatWithCursor(text, opts);
+async function format(text, options) {
+  const { formatted } = await formatWithCursor(text, {
+    ...options,
```

Comment:
I don't think we should use spread syntax for this because it is little expensive. Can we use `options.cursorOffset = -1`?

Reply:
I don't think so, this option is user input, we shouldn't modify it. It can also be `undefined`.

Reply:
I don't think clone an object is that expensive.

Reply:
Okay, we can keep it.